tests/h/cli/commands/authclient_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+
+from h import models
+from h.cli.commands import authclient as authclient_cli
+
+
+class TestAddCommand(object):
+
+    def test_it_creates_an_authclient(self, cli, cliconfig, db_session):
+        authclient = self._add_authclient(cli, cliconfig, db_session)
+        assert authclient
+
+    def test_it_prints_the_client_id_and_secret(self, cli, cliconfig, db_session, echo):
```

Comment:
I added this test after the initial implementation incorrectly printed `None` as the client ID and secret because it called `authclient.{id, secret}` before the object had actually been saved to the DB. I'll check to see if the output can be read / match from `result` instead of having to patch echo.